autoload/go/alternate.vim

```diff
@@ -0,0 +1,36 @@
+" By default use edit to switch to the new alternate buffer
+if !exists("g:go_alternate_mode")
+	let g:go_alternate_mode = "edit"
+endif
+
+" By default create the test file if it does not exist
+if !exists("g:go_create_test_file")
+	let g:go_create_test_file  = 1
```

Comment:
Thing is, its not really creating a file, it just create a buffer with the name of the file. Didn't want to do that? Pretty simple to just delete the new buffer. I'm using the bang right  now so that if the current buffer isn't saved you can switch with bang, so it matches other commands and their behaviour. If you really wan't, I can remove the option, and make the bang create the files.